FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -170,6 +172,10 @@ extension MessageViewController {
                 isEmail: true
             )
             let decryptErrBlocks = decrypted.blocks.filter { $0.decryptErr != nil }
+            let decryptAttBlocks = decrypted.blocks.filter { $0.type == .plainAtt || $0.type == .encryptedAtt || $0.type == .decryptedAtt }
```

Comment:
I would suggest to have some computed property for MsgBlock and cover it with tests(not in scope of this PR)
```
extension MsgBlock {
    var isAttributeBlock: Bool {
        type == .plainAtt || type == .encryptedAtt || type == .decryptedAtt
    }
}
```

Reply:
It would be called `isAttachmentBlock`. But this is rather minor as you say